plugins/connection/podman.py

```diff
@@ -141,9 +141,9 @@ def _connect(self):
         super(Connection, self)._connect()
         rc, self._mount_point, stderr = self._podman("mount")
         if rc != 0:
-            display.v("Failed to mount container %s: %s" % (self._container_id, stderr.strip()))
+            display.vvvv("Failed to mount container %s: %s" % (self._container_id, stderr.strip()))
         elif not os.listdir(self._mount_point.strip()):
-            display.v("Failed to mount container with CGroups2: empty dir %s" % self._mount_point.strip())
+            display.vvvv("Failed to mount container with CGroups2: empty dir %s" % self._mount_point.strip())
```

Comment:
FYI: The display on line 150 uses one `v` more.

Reply:
yes, stdout/stderr are more verbose everywhere